proposals/20220919-airflow_orchestration.md

```diff
@@ -19,13 +19,29 @@ Orchestrator
 We are moving Airflow from tfx/orchestration to tfx-addons. Native support for Airflow won't be provided in the near future.
 
 ## Project Implementation
-Move Airflow from tfx/orchestration to tfx-addons.
-
+1. Copy tfx/orchestration/airflow to tfx-addons/airflow.
```

Comment:
1. Please describe how the install will work
2. Please discuss any issues with testing in CI, given that this code will be independent of TFX

Reply:
So ... this is where it gets a bit interesting because the tutorials on the TF website need to reflect the deprecation too. It's a multi-part effort in which we need to first move the orchestrator to TFXA and THEN update the documentation, delete/remove dependencies from tfx/. We can't provide an accurate, line-by-line

Reply:
That's ok, just describe the issues and some options for dealing with them, and note that we haven't settled on a particular option yet.  At some point however, we will need a solid plan to do this so that we can be successful.